### PR TITLE
Update NLU test data

### PR DESCRIPTION
**Pull request by samsucik**
The test data is out-dated:
- chitchat examples are still listed as separate top-level intents
- `ask_when_next_event` is still in there even though it isn't used anymore

I fixed both issues. Note that `ask_when_next_event` still needs to be removed from the domain and from the training stories.

### CODE REVIEW

**Comment by samsucik** [comment]
Oh, no, I pushed some changes from the other PR 🤦  Gonna revert that.

File: tests/test_nlu.yml [hunk 1]
```diff
@@ -233,11 +233,7 @@ nlu:
     - what are you good at?
     - options
     - cool! can I do something else here?
-- intent: ask_when_next_event
-  examples: |
-    - what's the next rasa event
-    - is there a Rasa event in [San Francisco](location)
```
Comment (erohmensing) [inline tests/test_nlu.yml:239]: Maybe we should add these examples to the other intent where they live now? :)

**Review by erohmensing — APPROVED** [review]
Thanks, would you please create an issue for the outstanding to-do items you noticed with regards to this intent?

File: tests/test_nlu.yml (hunk 1 above)
Comment (samsucik) [reply tests/test_nlu.yml:239]: Oh, totally. Adding it now 🙂

**Comment by samsucik** [comment]
> would you please create an issue for the outstanding to-do items

Done 🙂

**Pull request merged**